WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -762,10 +767,16 @@ private int getCurrentPostStatusIndex() {
                 return 2;
             case PRIVATE:
                 return 3;
-            default:
-                // PUBLISHED, SCHEDULED, UNKNOWN
+            case TRASHED:
+            case UNKNOWN:
+            case PUBLISHED:
+            case SCHEDULED:
                 return 0;
         }
+        if (BuildConfig.DEBUG) {
+            throw new IllegalStateException("Missing switch case.");
```

Comment:
I think we should just use a lint rule for this. I assume we can't do it right away as there are probably some `switch` statements that don't have all the branches, but we could fix them and enable the lint rule afterwards.

Even if this is the approach we decide to take, let's not make it a part of an unrelated PR and instead try to add it for each `switch` statement as a single PR.

Reply:
We actually have this rule already enabled. This switch statement is part of the lint-baseline file, so it doesn't break the build. 
I'll remove the if, as you suggested;).